frontend-react/src/components/AuthElement.tsx

```diff
@@ -34,10 +34,23 @@ export const AuthElement = ({
     }, [requiredUserType, memberType]);
     // All the checks before returning the route
     useEffect(() => {
-        if (!oktaToken?.accessToken || !activeMembership) navigate("/login"); // Not logged in, needs to log in.
-        if (requiredUserType && !authorizeMemberType()) navigate("/"); // Not authorized as current member type
-        if (requiredFeatureFlag && !CheckFeatureFlag(requiredFeatureFlag))
-            navigate("/"); // Does not have feature flag enabled
+        if (!oktaToken?.accessToken || !activeMembership) {
+            navigate("/login");
+            return;
+        } // Not logged in, needs to log in.
+        if (requiredUserType && !authorizeMemberType()) {
+            navigate("/");
+            return;
+        } // Not authorized as current member type
+        if (requiredFeatureFlag && !CheckFeatureFlag(requiredFeatureFlag)) {
+            if (activeMembership.memberType === MemberType.PRIME_ADMIN) {
+                navigate("/admin/features");
```

Comment:
My favorite line in this PR:  as logged-in admins, if we try to navigate to a flagged route without the flag, it'll take us to the UI to add the flag. 
We could extend some functionality that conditionally checks if we pass a query like `?flagToAdd=[missing-flag]` and maybe use that to auto-populate the text field.